webapp/src/main/webapp/vue-apps/CallButton/components/JitsiMeetButton.vue

```diff
@@ -83,7 +84,7 @@ export default {
       } else if (this.callState === 'started' || this.callState === 'leaved') {
         return '#fb8e18';
       } else {
-        return '';
+        return '#476A9C';
```

Comment:
If the call is not started, the button should have primaryColor, not an hardcoded color